lib/bamboo/plug/sent_email_api/sent_email_api_plug.ex

```diff
@@ -3,12 +3,13 @@ defmodule Bamboo.SentEmailApiPlug do
   alias Bamboo.SentEmail
 
   @moduledoc """
-  A plug that can be used in your router to expose delivered emails over a
-  JSON API. This is useful for an integration test runner that doesn't have
+  A plug that to expose delivered emails over a JSON API.
```

Comment:
Extra word here?

Reply:
Thanks for catching that mistake. I'll get it updated.